includes/api/ApiQueryUserContributions.php

```diff
@@ -146,38 +128,24 @@ private function prepareQuery() {
 		$tables = array( 'page', 'revision' ); // Order may change
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app